osgp/platform/osgp-domain-core/src/main/java/org/opensmartgridplatform/domain/core/valueobjects/smartmetering/AbstractPushSetup.java

```diff
@@ -171,6 +175,10 @@ public boolean hasCommunicationWindow() {
         return this.communicationWindow != null;
     }
 
+    // suppress warning about returning null. With this method a null return is currently used to show that the list
+    // that is used isn't actually initialized. This means that there isn't really an absence of objects to put in
+    // the list, but rather that the object to put into it couldn't be made.
+    @SuppressWarnings("squid:S1168")
```

Comment:
I don't think we should suppress this warning, and return an empty list.
Write unit tests for all the code calling this method to make sure everything keeps on working ok.